apps/evalite-ui/app/routes/eval.$name.tsx

```diff
@@ -169,9 +170,17 @@ export default function Page() {
                 data={history}
                 onDotClick={({ date }) => {
                   if (date === mostRecentDate) {
-                    setSearch({});
+                    debugger;
```

Comment:
There's an existing bug where you have to click twice on a dot if none of the scores change from the lastest run. it's not an issue with remix or tanstack router as far as I can tell and It'll need more investigation.